### ISSUE TITLE
AlfDocumentList.js maybe wrong extend in the doc

### ISSUE THREAD

**Issue by RedRaccoon** [Alfresco/Aikau#1098]
Hello,

As you can see in the [AlfDocumentList.js in the doc](https://dev.alfresco.com/resource/docs/aikau-jsdoc/AlfDocumentList.js.html) 

```
 * @module alfresco/documentlibrary/AlfDocumentList
 * @extends module:alfresco/lists/AlfFilteredList
 * @author Dave Draper
 */
define(["dojo/_base/declare",
        "alfresco/lists/AlfFilteredList",
        "allInclude"],
        function(declare, AlfSortablePaginatedList, allInclude) {

   return declare([AlfSortablePaginatedList], {
```

The AlfDocumentList is an extension of AlfilteredList, even if it's 

> AlfSortablePaginatedList

in variable it's still AlfFilteredList who is using for the extend

but as I had an error I checked in my own AlfDocumentList.js in my aikau jar (it's 1.0.25.2 version)

```
 * @module alfresco/documentlibrary/AlfDocumentList
 * @extends module:alfresco/lists/AlfSortablePaginatedList
 * @author Dave Draper
 */
define(["dojo/_base/declare",
        "alfresco/lists/AlfSortablePaginatedList",
        "allInclude"],
        function(declare, AlfSortablePaginatedList, allInclude) {

   return declare([AlfSortablePaginatedList], {
```

So maybe the AlfDocumentList used to extend from afiltered or maybe it's wrong extend

thank you

**Comment by draperd** [comment]
Yes... AlfDocumentList used to extend AlfSortablePaginatedList (before AlfFilteredList existed). However, when we introduced the AlfFilteredList we made AlfDocumentList extend it instead. This didn't change the behaviour of AlfDocumentList, it just gave it more capabilities. You're correct in pointing out that we've  failed to update the variable name - but as you rightly point out, it's still the AlfFilteredList even if it's assigned to a variable called AlfSortablePaginatedList.

Beyond changing the variable name, is there anything else you're looking for us to do here?
